### ISSUE TITLE
Potentially Infinite Loop

### ISSUE THREAD

**Issue by joshgoebel** [taufik-nurrohman/highlight.jit.js#1]
https://github.com/taufik-nurrohman/highlight.jit.js/blob/f4d96e0e3a7e014cfe760836e05ba575270b1d83/highlight.jit.js#L32

Did you test what happens if a 404 is hit or if load completes successfully yet the JS file there is invalid or does not properly load?

Putting an empty JS file there would be the quickest way to test for this.

**Comment by taufik-nurrohman** [comment]
No, I did not test it. Maybe it would be better if I use AJAX, to get the response status and MIME type next time. And so making the language register becomes inline:

``` .js
if (
    200 === xhr.status &&
    ['application/javascript', 'text/javascript', ...].indexOf(xhr.getResponseHeader('Content-Type')) > -1
) {
    // eval(xhr.response);
    // or...
    let script = document.createElement('script');
    script.textContent = xhr.response;
    document.body.appendChild(script);
    hljs.highlightBlock(block);
}
```

**Comment by taufik-nurrohman** [comment]
> Putting an empty JS file there would be the quickest way to test for this.

Will do make a test folder then.
